samples/benchmarks/coremark/README.rst

```diff
@@ -119,7 +119,7 @@ CONFIG_APP_MODE_FLASH_AND_RUN - Start CoreMark sample automatically after flashi
    Otherwise, it will wait for the button press.
 
 .. note::
-   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the radio core and the PPR core.
+   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core.
```

Comment:
`... for the nRF54H20 PPR core.`?

Reply:
I wouldn't specify SoC. Right now we don't have ppr on any other SoC but if we had we would follow with the same approach.